automation/src/test/java/org/greenplum/pxf/automation/features/hive/HiveOrcTest.java

```diff
@@ -106,6 +138,46 @@ private void preparePxfHiveSmallData() throws Exception {
         gpdb.copyData(exTable, gpdbNativeTable);
     }
 
+   private void prepareOrcLargeData(int numRows) throws Exception {
+        hiveOrcLargeDataTable = new HiveTable("hive_orc_large_data", HIVE_ORC_LARGE_DATA_COLS);
+        hiveOrcLargeDataTable.setStoredAs(ORC);
+        hive.createTableAndVerify(hiveOrcLargeDataTable);
+        Configuration configuration = new Configuration();
+        configuration.set("orc.overwrite.output.file", "true");
+        TypeDescription schema = TypeDescription.fromString(ORC_LARGE_DATA_TYPE);
+        String fileName = String.format("large_data_%d_rows.orc", numRows);
+        largeOrcDataFile = dataTempFolder + "/" + fileName;
+        Writer writer = OrcFile.createWriter(new Path(largeOrcDataFile),
+                OrcFile.writerOptions(configuration).setSchema(schema));
+
+        VectorizedRowBatch batch = schema.createRowBatch();
+        LongColumnVector col1 = (LongColumnVector) batch.cols[0];
+        BytesColumnVector col2 = (BytesColumnVector) batch.cols[1];
+
+        for (int i = 0; i < numRows; i++) {
+            int row = batch.size++;
+
+            byte[] b1 = String.format("row%d", i).getBytes(StandardCharsets.UTF_8);
```

Comment:
```suggestion
            byte[] bytes = String.format("row%d", i).getBytes(StandardCharsets.UTF_8);
```
not sure why it was called b1